qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -142,7 +142,17 @@ def main(self,argsOverride=None,bitcoinConfDict=None,wallets=None):
                           help="Don't stop bitcoinds after the test execution")
         parser.add_option("--srcdir", dest="srcdir", default=os.path.normpath(os.path.dirname(os.path.realpath(__file__))+"/../../../src"),
                           help="Source directory containing bitcoind/bitcoin-cli (default: %default)")
-        parser.add_option("--tmpdir", dest="tmpdir", default=tempfile.mkdtemp(prefix="test"),
+
+
+        from os.path import basename
+        import string
+        from sys import argv
```

Comment:
I rather add this `import` statement on top of the file.

Reply:
Done.